fabric-registry-sync-v0/src/main/java/net/fabricmc/fabric/api/event/registry/FabricRegistryBuilder.java

```diff
@@ -67,7 +67,7 @@
 	 * @param <T> The type stored in the Registry
 	 * @return An instance of FabricRegistryBuilder
 	 */
-	public static <T> FabricRegistryBuilder<T, SimpleRegistry<T>> createSimple(class_5321<Registry<T>> registryKey, Lifecycle lifecycle) {
+	public static <T> FabricRegistryBuilder<T, SimpleRegistry<T>> createSimple(RegistryKey<Registry<T>> registryKey, Lifecycle lifecycle) {
```

Comment:
Can't `Lifecycle.stable()` be a default?

Reply:
Oh, I can't read apparently

Reply:
People should decide the lifecycle themselves.

Reply:
> People should decide the lifecycle themselves.

Why... `Lifecycle.experimental()` should not be used except in truly experimental circumstances (which are exceptional). It causes a warning message to pup up for the player.